recipes_source/quantization.rst

```diff
@@ -81,7 +81,7 @@ The full documentation of the `quantize_dynamic` API call is `here <https://pyto
 3. Post Training Static Quantization
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-This method converts both the weights and the activations to 8-bit integers beforehand so there won't be on-the-fly conversion on the activations during the inference, as the dynamic quantization does, hence improving the performance significantly.
+This method converts both the weights and the activations to 8-bit integers beforehand so there won’t be on-the-fly conversion on the activations during the inference, as the dynamic quantization does. While post-training static quantization can significantly enhance inference speed and reduce model size, this method may degrade the original model's performance.
```

Comment:
makes sense, can you clarify a little bit more, e.g.:
```
this method may degrade the original model's accuracy more compared to dynamic quantization. 
```

Reply:
Like it

Reply:
Hi @jerryzh168, I've just pushed a few changes, please let me know if you have any other suggestions. Thank you!

Reply:
@jmarintur "Converting weights and activation functions to 8-bit integers can slightly alter the network's behavior and activation responses, leading to such variations in accuracy." this is true for any quantization method, so probably better to remove I feel